### DISCUSSION TITLE
Any plan to support Slint UI?

### DISCUSSION THREAD

**Discussion by cokkiy** [plotters-rs/plotters#521] [Ideas]
Is there any plan to support [Slint](https://github.com/slint-ui/slint/) as a widget? Slint is an excellent native UI toolkit for Rust but lacks a plotter widget, so how can this plotter be used in Slint?

**Comment by JuliDi** [comment]
I'm interested in this, too, but have very little knowledge, yet, of what would need to be done to support slint. Could anyone with more experience provide a brief overview, if possible? This might be very interesting for embedded systems.

**Reply by JuliDi** [reply]
Seems like slint's plotter example already uses plotters: https://github.com/slint-ui/slint/blob/2871fadd5cc5e0efd760f106b2cffe9cda8d1f47/examples/plotter/main.rs#L4
